src/server/service/scenarios/db.js

```diff
@@ -0,0 +1,53 @@
+const { sql } = require('../../util/sqlHelpers');
+const { withClient, withClientTransaction } = require('../../util/db');
+
+exports.getScenario = async function getScenario(scenarioId) {
+    return withClient(async client => {
+        const results = await client.query(
+            sql` SELECT * FROM scenario WHERE id = ${scenarioId};`
+        );
+        return results.rows[0];
+    });
```

Comment:
this pattern repeats a lot too...

what about a thing to get rid of the `withClient` wrappings...

util/db.js:
```js
exports.query = (...args) => exports.withClient(client => client.query(...args));
```

lets you rewrite this:

```js
const results = await client.query(
  sql` SELECT * FROM scenario WHERE id = ${scenarioId};`
);
return results.rows[0];
```

if you want to get super terse -- 

```js
exports.getScenario = async scenarioId =>
    (await query(sql` SELECT * FROM scenario WHERE id = ${scenarioId};`))
        .rows[0];
```

Reply:
Really cool!